pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/argoproj.io/v1alpha1/Workflow/customizations.yaml

```diff
@@ -0,0 +1,449 @@
+              finishedAt = statusItems[i].status.finishedAt
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.estimatedDuration ~= nil then
+              estimatedDuration = estimatedDuration + statusItems[i].status.estimatedDuration
+            end
+
+            if statusItems[i].status ~= nil and statusItems[i].status.tmpGeneration ~= nil then
+              generation = statusItems[i].status.tmpGeneration
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.compressedNodes ~= nil and statusItems[i].status.compressedNodes ~= '' then
+              compressedNodes = statusItems[i].status.compressedNodes
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.nodes ~= nil then
+              nodes = statusItems[i].status.nodes
+            end          
+            if statusItems[i].status ~= nil and statusItems[i].status.offloadNodeStatusVersion ~= nil and statusItems[i].status.offloadNodeStatusVersion ~= '' then
+              offloadNodeStatusVersion = statusItems[i].status.offloadNodeStatusVersion
+            end   
+            if statusItems[i].status ~= nil and statusItems[i].status.storedTemplates ~= nil then
+              storedTemplates = statusItems[i].status.storedTemplates
+            end     
+            if statusItems[i].status ~= nil and statusItems[i].status.persistentVolumeClaims ~= nil then
+              persistentVolumeClaims = statusItems[i].status.persistentVolumeClaims
+            end   
+            if statusItems[i].status ~= nil and statusItems[i].status.outputs ~= nil then
+              outputs = statusItems[i].status.outputs
+            end   
+            if statusItems[i].status ~= nil and statusItems[i].status.storedWorkflowTemplateSpec ~= nil then
+              storedWorkflowTemplateSpec = statusItems[i].status.storedWorkflowTemplateSpec
+            end   
+            if statusItems[i].status ~= nil and statusItems[i].status.synchronization ~= nil then
+              synchronization = statusItems[i].status.synchronization
+            end   
+            if statusItems[i].status ~= nil and statusItems[i].status.artifactRepositoryRef ~= nil then
+              artifactRepositoryRef = statusItems[i].status.artifactRepositoryRef
+            end   
+            if statusItems[i].status ~= nil and statusItems[i].status.artifactGCStatus ~= nil then
+              artifactGCStatus = statusItems[i].status.artifactGCStatus
+            end 
+
+
+            if statusItems[i].status ~= nil and statusItems[i].status.progress ~= nil then
+              if progress == '' then
+                progress = progress..statusItems[i].clusterName..'='..statusItems[i].status.progress
+              else
+                progress = progress..','..statusItems[i].clusterName..'='..statusItems[i].status.progress
+              end
+            end
```

Comment:
Progress in N/M format. N is number of task complete. M is number of tasks.[ref](https://github.com/argoproj/argo-workflows/blob/master/pkg/apis/workflow/v1alpha1/progress.go#L9)